src/variables.jl

```diff
@@ -119,6 +119,37 @@ true
 """
 owner_model(v::AbstractVariableRef) = v.model
 
+"""
+    struct VariableNotOwned{V <: AbstractVariableRef} <: Exception
+        variable::V
+    end
+
+The variable `variable` was used in a model different to
+`owner_model(variable)`.
+"""
+struct VariableNotOwned{V <: AbstractVariableRef} <: Exception
+    variable::V
+end
+
+"""
+    check_belongs_to_model(model::AbstractModel, func::AbstractJuMPScalar)
```

Comment:
The order of the parameters should be switched to match the order of words in the function name, i.e., check that the variable belongs to the model.